plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -1999,7 +1999,10 @@ int amdgpu_plugin_resume_devices_late(int target_pid)
 	args.op = KFD_CRIU_OP_RESUME;
 	pr_info("Calling IOCTL to start notifiers and queues\n");
 	if (kmtIoctl(fd, AMDKFD_IOC_CRIU_OP, &args) == -1) {
-		pr_perror("restore late ioctl failed");
+		if (errno == ESRCH)
+			pr_info("Pid %d has no kfd process info\n", target_pid);
```

Comment:
We don't print error here, but we still return -1, I feel that this behavior is kind of inconsistent? Do I miss something?

Reply:
criu "ignores" this error code: https://github.com/checkpoint-restore/criu/blob/criu-dev/criu/cr-restore.c#L2487

I think @Snorch is right, we have to return 0 in this case.

Reply:
Ok, then let's do it:

https://github.com/checkpoint-restore/criu/pull/2343